user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -656,7 +699,7 @@ define([
 
     fetchData: function (path, callback) {
       var myself = this,
-          url = this.getFileListRequest(FileBrowser.encodePathComponents(path == null ? ":" : Encoder.encodeRepositoryPath(path))),
+          url = this.getFileListRequest(path == null ? ":" : path.replaceAll(':','~').replaceAll('/',':')),
```

Comment:
See suggestion below regarding `_encodeGenericPath`.

Also, can `path` really ever be `null` here?

Reply:
from every flow I've seen, it will always be "/"... I persisted some logic I didn't fully understand fearing that I could be introducing a regression to something somewhere, maybe a plugin or customer custom solution or something?

Reply:
Ok. See my response in referenced comment.